azext_edge/edge/params.py

```diff
@@ -646,62 +646,41 @@ def load_iotops_arguments(self, _):
             help="Asset name.",
         )
         context.argument(
-            "endpoint",
-            options_list=["--endpoint"],
-            help="Asset endpoint name.",
-        )
-        context.argument(
-            "custom_attributes",
-            options_list=["--custom-attribute", "--attr"],
-            help="Space-separated key=value pairs corresponding to additional custom attributes for the asset.",
-            nargs="+",
-            arg_group="Additional Info",
-            action="extend",
-        )
-        context.argument(
-            "custom_location_name",
-            options_list=["--custom-location", "--cl"],
-            help="Custom location used to associate asset with cluster.",
-        )
-        context.argument(
-            "custom_location_resource_group",
-            options_list=["--custom-location-resource-group", "--clg"],
-            help="Resource group for custom location.",
-        )
-        context.argument(
-            "custom_location_subscription",
-            options_list=["--custom-location-subscription", "--cls"],
-            help="Subscription Id for custom location. If not provided, asset subscription Id will be used.",
+            "endpoint_profile",
+            options_list=["--endpoint-profile", "--ep"],
+            help="Asset endpoint profile name.",
         )
         context.argument(
-            "cluster_name",
-            options_list=["--cluster", "-c"],
-            help="Cluster to associate the asset with.",
+            "instance_name",
+            options_list=["--instance"],
+            help="Instance name to associate the created asset with."
         )
         context.argument(
-            "cluster_resource_group",
-            options_list=["--cluster-resource-group", "--cg"],
-            help="Resource group for cluster.",
+            "instance_resource_group",
+            options_list=["--instance-resource-group", "--ig"],
+            help="Instance resource group. If not provided, asset resource group will be used."
         )
         context.argument(
-            "cluster_subscription",
-            options_list=["--cluster-subscription", "--cs"],
-            help="Subscription Id for cluster. If not provided, asset subscription Id will be used.",
+            "instance_subscription",
+            options_list=["--instance-subscription", "--is"],
+            help="Instance subscription id. If not provided, asset subscription id will be used."
         )
         context.argument(
-            "asset_type",
-            options_list=["--asset-type", "--at"],
-            help="Asset type.",
+            "custom_attributes",
+            options_list=["--custom-attribute", "--attr"],
```

Comment:
Note to talk about.

Reply:
This is the same format as before - just moved a bit down cause github. But we can talk about this.

Reply:
Topic intended for additionalConfiguration.